<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APTDemodGUI</class>
 <widget class="RollupWidget" name="APTDemodGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>APT Demodulator</string>
  </property>
  <property name="statusTip">
   <string>APT Demodulator</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="powLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="deltaFrequencyLabel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Df</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDialZ" name="deltaFrequency" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Demod shift frequency from center in Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deltaUnits">
        <property name="text">
         <string>Hz </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="channelPowerLayout">
        <item>
         <widget class="QLabel" name="channelPower">
          <property name="toolTip">
           <string>Channel power</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="channelPowerUnits">
          <property name="text">
           <string> dB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="levelLayout">
      <item>
       <widget class="QLabel" name="channelPowerMeterUnits">
        <property name="text">
         <string>dB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LevelMeterSignalDB" name="channelPowerMeter" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Level meter (dB) top trace: average, bottom trace: instantaneous peak, tip: peak hold</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="sliderLayout">
      <item>
       <widget class="QLabel" name="rfBWLabel">
        <property name="text">
         <string>BW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rfBW">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>RF bandwidth</string>
        </property>
        <property name="minimum">
         <number>300</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rfBWText">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>40.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fmDevLabel">
        <property name="text">
         <string>Dev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fmDev">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Frequency deviation</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>170</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fmDevText">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>17.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="ButtonSwitch" name="startStop">
        <property name="toolTip">
         <string>Start/stop decoding</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/stop.png</normalon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="showSettings">
        <property name="toolTip">
         <string>Show settings dialog</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/listing.png</normaloff>:/listing.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="resetDecoder">
        <property name="toolTip">
         <string>Reset decoder (clears current image)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/bin.png</normaloff>:/bin.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveImage">
        <property name="toolTip">
         <string>Save image to disk</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/save.png</normaloff>:/save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomIn">
        <property name="toolTip">
         <string>Zoom in to image</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/zoomin.png</normaloff>:/zoomin.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomOut">
        <property name="toolTip">
         <string>Zoom out from image</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/zoomout.png</normaloff>:/zoomout.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="zoomAll">
        <property name="toolTip">
         <string>Zoom image to fit window</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/zoomall.png</normaloff>:/zoomall.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="channels">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Which channels from the image to display</string>
        </property>
        <item>
         <property name="text">
          <string>Both</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="cropNoise">
        <property name="toolTip">
         <string>Crop noise from top and bottom of image</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/apt/icons/cropnoise.png</normaloff>:/apt/icons/cropnoise.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="denoise">
        <property name="toolTip">
         <string>Apply denoise filter to the image</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/apt/icons/denoise.png</normaloff>:/apt/icons/denoise.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="linear">
        <property name="toolTip">
         <string>Apply linear equalisation to the image</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/linear.png</normaloff>:/linear.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="histogram">
        <property name="toolTip">
         <string>Apply histogram equalisation to the image</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/dsb.png</normaloff>:/dsb.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="precipitation">
        <property name="toolTip">
         <string>Overlay precipitation</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/apt/icons/precipitation.png</normaloff>:/apt/icons/precipitation.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="flip">
        <property name="toolTip">
         <string>Satellite pass direction (flips image)</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/arrow_down.png</normaloff>
          <normalon>:/arrow_up.png</normalon>:/arrow_down.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="imageContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>431</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Received Image</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutTable">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="channelTypeHorizontalLayout">
      <item>
       <widget class="QLabel" name="channelALabel">
        <property name="text">
         <string>Channel A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelBLabel">
        <property name="text">
         <string>Channel B</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="image">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>350</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelMeterSignalDB</class>
   <extends>QWidget</extends>
   <header>gui/levelmeter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>deltaFrequency</tabstop>
  <tabstop>rfBW</tabstop>
  <tabstop>fmDev</tabstop>
  <tabstop>startStop</tabstop>
  <tabstop>showSettings</tabstop>
  <tabstop>resetDecoder</tabstop>
  <tabstop>saveImage</tabstop>
  <tabstop>channels</tabstop>
  <tabstop>cropNoise</tabstop>
  <tabstop>denoise</tabstop>
  <tabstop>linear</tabstop>
  <tabstop>histogram</tabstop>
  <tabstop>precipitation</tabstop>
  <tabstop>flip</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
